imports/api/employee.js

```diff
@@ -0,0 +1,209 @@
+import {Meteor} from "meteor/meteor";
```

Comment:
Please either use eslint plugin or run test to make sure the code follow the lint rule.